R/ppc-distributions.R

```diff
@@ -528,6 +545,150 @@ ppc_violin_grouped <- function(y, yrep, group, ..., probs = c(0.1, 0.5, 0.9),
     bayesplot_theme_get()
 }
 
+#' @export
+#' @param pit An optional vector of probability integral transformed values for
```

Comment:
please double check if all arguments are documented

Reply:
The remaining arguments are documented in [man-roxygen/args-pit-ecdf.R](https://github.com/TeemuSailynoja/bayesplot/blob/issue-267-ecdf-plots-with-confidence-bands/man-roxygen/args-pit-ecdf.R) now that the `adj_method` parameter has been updated to `interpolate_adj`.